src/NDS.cpp

```diff
@@ -823,70 +826,72 @@ u32 RunFrame()
 {
     FrameStartTimestamp = SysTimestamp;
 
-    if (!Running) return 263; // dorp
-    if (CPUStop & 0x40000000) return 263;
-
-    GPU::StartFrame();
-
-    while (Running && GPU::TotalScanlines==0)
+    bool didAnything = false;
+    if (Running && !(CPUStop & 0x40000000))
```

Comment:
how is this different than the original behaviour? If I'm not wrong you could just increment FramesSinceBoot at the beginning of RunFrame.

Reply:
One reason for doing it that way was that it should be 0 on the first frame of emulation. If I simply incremented it at the start of the frame, the RTC would think it's 1/60th of a second later than it is. This could be solved by initializing it to -1 instead of 0, if FramesSinceBoot were a signed value.

Also this change is for the other counters as well, NumFrames and NumLagFrames. I guess these could have a similar fix with initializing to -1. But it seems simpler to me to do it this way rather than make those changes, and by having the counter increments at the end it's more obvious when reading the code that they are counting completed frames rather than started frames.